receiver/selreceiver/internal/selinputoperator/input.go

```diff
@@ -0,0 +1,67 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package selInputOperator // import "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/input/stdin"
+
+import (
+	"context"
+	"sync"
+
+	"go.uber.org/zap"
+
+	"github.com/bougou/go-ipmi"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/entry"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator/helper"
+)
+
+// Input is an operator that reads input from stdin
+type Input struct {
+	helper.InputOperator
+	wg         sync.WaitGroup
+	cancel     context.CancelFunc
+	ipmiClient *ipmi.Client
+}
+
+// Start will start generating log entries.
+func (i *Input) Start(_ operator.Persister) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	i.cancel = cancel
+
+	i.wg.Add(1)
+	go func() {
+		defer i.wg.Done()
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			default:
+			}
+
+			selList, err := i.ipmiClient.GetSELEntries(ctx, 0)
```

Comment:
since this loop is continuous, will this tend to emit the same events or does retrieving them clear the existing set? should there be a timer for the loop?